Build/libHttpClient.Apple.C/Dummy/Dummy.h

```diff
@@ -0,0 +1,12 @@
+//
+//  Dummy.h
```

Comment:
from your name from dummy.h & dummy.m.

also can you give the PR a good title?  Its not clear what your trying to do

Reply:
Hey Jason. I'll clean this up. Matt asked me to create a pr that seeks to solve a problem that I have now added a description of.

Reply:
Ah, cool.  Thanks